src/components/button/Button.scss

```diff
@@ -54,18 +55,45 @@
     font-size: 13px;
     line-height: 16px;
     padding: 0 4px;
+    &.withIcon {
+      padding-right: 8px;
+      > span {
+        margin-left: 2px;
+      }
+    }
+    &.iconOnly {
+      padding: 0 3px;
+    }
   }
   &.medium {
     height: 32px;
     font-size: 14px;
     line-height: 17px;
     padding: 0 8px;
+    &.withIcon {
+      padding-right: 12px;
+      > span {
+        margin-left: 4px;
+      }
+    }
+    &.iconOnly {
+      padding: 0 3px;
+    }
   }
   &.large {
     height: 40px;
     font-size: 16px;
     line-height: 19px;
     padding: 0 16px;
+    &.withIcon {
```

Comment:
are values in `withIcon` and `iconOnly` for `large` class should be the same as `medium` class?